docs/ui/prepare-migration/datastream-tuning.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: default
+title: Tune Datastream (Optional)
+parent: Prepare Migration Page
+grand_parent: SMT UI
+nav_order: 4
+---
+
+# Tune Datastream (Optional)
+{: .no_toc }
+
+In case of minimal downtime migration, the Datastream jobs launched by the Spanner Migration Tool can be **optionally** tuned. Tuning refers to tweaking these parameters to run Datastream is a custom configuration.
+
+## Tuning use cases
+
+SMT by default launches datastream 50 backfill tasks and 5 cdc tasks.
+
+Some use cases when the user would want to tweak these:
+- Reduce load during backfill on source database
+- Support more table processing in parallel for change events
+
+
+{: .highlight }
+
+To tune datastream, first specify the target database in the 'Configure Spanner Database' step. This enables the configure button for the remaining steps.
+
+![](https://services.google.com/fh/gumdrop/preview/misc/datastream-tuning.png)
+
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+
+{: .highlight }
+
+SMT exposes the maxConcurrentBackfillTasks and maxConcurrentCdcTasks for tuning. Please reach out to us if you have a use-case that is not satisfied by the provided configurations.
+
+{: .highlight }
+Increasing the number of tasks does not speedup the processing of individual tables. Since each task is dedicated to processing a single table, adding more tasks will not enhance the processing speed of a single table. Essentially, the number of tasks determines the maximum number of tables that can be processed simultaneously.
```

Comment:
I think this can just be inline text? Both of these as highlights looks a bit weird on the docs?
https://screenshot.googleplex.com/5uWdpDJf6x4gXiH